requirements.txt

```diff
@@ -1,9 +1,10 @@
+astroid==2.0.4
+dj-database-url==0.5.0
+django-heroku==0.3.1
+gunicorn==19.9.0
+psycopg2==2.7.5
+psycopg2-binary==2.7.5
 coverage==4.5.1
-Django==2.1.1
-django-cors-middleware==1.3.1
-django-extensions==2.1.2
-djangorestframework==3.8.2
-djangorestframework-jwt==1.11.0
```

Comment:
I don't understand this change, these are important libraries and frameworks you've removed. Why?

Reply:
There are available in the current requirements.txt